# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sway/default-t2-background.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sway-1.9/config.in.vanilla	2024-05-24 16:35:42.737887194 +0200
+++ sway-1.9/config.in	2024-05-24 16:36:00.009885530 +0200
@@ -23,7 +23,7 @@
 ### Output configuration
 #
 # Default wallpaper (more resolutions are available in @datadir@/backgrounds/sway/)
-output * bg @datadir@/backgrounds/sway/Sway_Wallpaper_Blue_1920x1080.png fill
+output * bg @datadir@/backgrounds/sway/t2sde.svg fill #19446f
 #
 # Example configuration:
 #
